tyk-pro/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: Deploys Tyk Dashboard, assumes that MongoDB and Redis have already been installed
 name: tyk-pro
-version: 0.8.0
+version: 0.8.1
```

Comment:
I forgot to bump the version in #105 but with these many changes, shouldn't it be a minor version?

Reply:
@excieve recommended thiS version update. But I don't mind a minor version update either.